Update INSTALL.md

dwasss opened the pull request:
Added updating requirements output dependencies.

dwasss committed:
Update INSTALL.md

Added updating requirements output dependencies.

micheloosterhof commented:
Hi. The optional requirements are options, so we shouldn't add them to the default list.
They can be installed on demand by people who want that specific functionality.

dwasss commented:
Yes, but people might not think to update the output requirements when updating cowrie. It happened to me :) so that is why I am suggesting it in the install guide as a reminder. Maybe I can put some text saying that it is optional?

micheloosterhof commented:
Sure, that would be a better solution.

dwasss committed:
Update INSTALL.md

Updated text to include info about upgrading optional output plugin dependencies

dwasss commented:
OK, just updated this.
